addon/ng2/utilities/get-dependent-files.ts

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+import * as fs from 'fs';
+import * as ts from 'typescript';
+import * as glob from 'glob';
+import * as path from 'path';
+import * as denodeify from 'denodeify';
+
+import { Promise } from 'es6-promise';
+
+/**
+ * Declare an Interface where key is a string and value is either a string or a number.
```

Comment:
I don't quite follow how this sentence relates to `ModuleImport`.

In general, a good comment should talk about what the object or interface _is_ instead of what's inside it.